.github/actions/docker-build-push/action.yml

```
@@ -53,7 +53,7 @@ runs:
         registry: ${{ inputs.registry }}
     - name: Calculate Branch and Base Names
       id: refs
-      uses: chanzuckerberg/github-actions/.github/actions/get-github-ref-names@get-github-ref-names-v1.5.0
+      uses: chanzuckerberg/github-actions/.github/actions/get-github-ref-names@get-github-ref-names-v1.5.2
```

```suggestion
      uses: chanzuckerberg/github-actions/.github/actions/get-github-ref-names@get-github-ref-names-v1.6.0
```
something about the releases+tagging are broken in this repo.. https://github.com/chanzuckerberg/github-actions/releases/tag/get-github-ref-names-v1.6.0 but for now v1.6.0 works fine here